authors/apps/articles/models.py

```diff
@@ -0,0 +1,71 @@
+from django.db import models
+from django.contrib.auth import get_user_model
+from django.utils.text import slugify
+
+from cloudinary.models import CloudinaryField
+from cloudinary import CloudinaryImage
+
+from authors.apps.profiles.models import Profile
+
+User = get_user_model()
+
+
+class Articles(models.Model):
```

Comment:
Hello @matthenge ,
The model should be singular in form since it is a representation of every an individual entity, not a container of entities. You could rename the model to `Article` instead of `Articles`